modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/configurators/TargetPlatformRootProjectConfigurator.java

```diff
@@ -109,15 +111,30 @@ private TargetPlatformRootProjectConfigurator() {
 	private void _addDependenciesTargetPlatformBoms(
 		Project project, String targetPlatformVersion) {
 
+		Matcher dxpVersion = _dxpVersionPattern.matcher(targetPlatformVersion);
```

Comment:
Variable names should end with the data type name. In this method there's only one `Matcher`, so `dxpVersion` can be renamed to `matcher`.